rt/mlc/TESTDC1.MLC

```diff
@@ -439,14 +440,20 @@ CVTTOHEX2 RT1   CCE,=VD(CVTTOHEX),=AL8(CVTTOHEX) RPI 1044 DIFF ADDR
 &AVAL    SETA  2147483647
 &CHEX    SETC  A2X(&AVAL)
          RT1   CCE,=C'&CHEX',=C'7FFFFFFF'
+* Next test succeeds; will fail when 2147483648 flagged as error #509
          RT1   CCE,=A(2147483648),=X'80000000'      TEST SETA OVERFLOW
-&AVAL    SETA  2147483648
-&CHEX    SETC  A2X(&AVAL)                     TEST NEG A2X
-         RT1   CCE,=C'&CHEX',=C'80000000'
-         RT1   CCE,=A(2147483649),=X'80000001'      
-&AVAL    SETA  2147483649
-&CHEX    SETC  A2X(&AVAL)
-         RT1   CCE,=C'&CHEX',=C'80000001'
+* Before SETA; now fails, as it should                      #509
```

Comment:
I understand these tests needed to be commented out.
Yet it is - in my opinion - still useful to have failing edge cases in the test suite.
Should they go into a separate test program that is expected to fail RC=8?
If yes, could you please the commented-out code into an issue that covers the work?